File: Modul 4/Bahasa C/PRAK403-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h>
int main() {
    int angka1, angka2;
    
    printf("Input\n");
    scanf("%d %d", &angka1, &angka2);
    printf("\nOutput\n");

    if (angka1 < angka2) {
        for (int i = angka1; i <= angka2; i++) {
            printf("%d %d", i, angka1 + angka2 - i);
            if (i < angka2){
                printf(" - ");
            }
        }
    } else if (angka1 > angka2) {
        for (int i = angka1; i >= angka2; i--) {
            printf("%d %d", i, angka1 + angka2 - i);
            if (i > angka2){
                printf(" - ");
            }
        }
    } else {
        printf("%d %d", angka1, angka2);
    }
    return 0;
}

File: Modul 4/Bahasa Python/PRAK403-2310817120001-DinaIzzatiElfadheya.py
print("Input")
angka1, angka2 = map(int, input().split())
print("\nOutput")

if angka1 < angka2:
    for i in range(angka1, angka2 + 1):
        print(i, angka1 + angka2 - i, end=" - " if i < angka2 else "")
elif angka1 > angka2:
    for i in range(angka1, angka2 - 1, -1):
        print(i, angka1 + angka2 - i, end=" - " if i > angka2 else "")
else:
    print(angka1, angka2, end=" ")